src/ProfilesList.json

```
@@ -270,6 +270,7 @@
   "Kritika30032002.json",
   "dellucifer.json",
   "shricodev.json",
+  "ZeeshanMukhtar1.json"
```

```suggestion
  "ZeeshanMukhtar1.json",
```
Add comma `,` after filename.